src/lib389/lib389/cli_base/__init__.py

```diff
@@ -409,7 +409,12 @@ def __len__(self):
 
 class StdErrFilter(logging.Filter):
```

Comment:
It's only partially related to this PR, but I get some weird empty errors if I specify an incorrect suffix during instance creation.

Web UI:
[Screencast from 02-13-2023 05:47:53 PM.webm](https://user-images.githubusercontent.com/5634892/218620629-f4d74044-6722-4397-ba43-97c3de472306.webm)


CLI debug mode:
```
DEBUG: ACTION: Creating dse.ldif
DEBUG: 
Traceback (most recent call last):
  File "/usr/sbin/dscreate", line 99, in <module>
    result = args.func(inst, log, args)
  File "/usr/lib/python3.10/site-packages/lib389/cli_ctl/instance.py", line 71, in instance_create
    if sd.create_from_inf(args.file):
  File "/usr/lib/python3.10/site-packages/lib389/instance/setup.py", line 562, in create_from_inf
    self.create_from_args(general, slapd, backends, self.extra)
  File "/usr/lib/python3.10/site-packages/lib389/instance/setup.py", line 698, in create_from_args
    self._install_ds(general, slapd, backends)
  File "/usr/lib/python3.10/site-packages/lib389/instance/setup.py", line 751, in _install_ds
    ds_suffix = normalizeDN(backends[0]['nsslapd-suffix'])
  File "/usr/lib/python3.10/site-packages/lib389/utils.py", line 504, in normalizeDN
    ary = ldap.explode_dn(dn.lower())
  File "/usr/lib64/python3.10/site-packages/ldap/dn.py", line 74, in explode_dn
    dn_decomp = str2dn(dn,flags)
  File "/usr/lib64/python3.10/site-packages/ldap/dn.py", line 48, in str2dn
    return ldap.functions._ldap_function_call(None,_ldap.str2dn,dn,flags)
  File "/usr/lib64/python3.10/site-packages/ldap/functions.py", line 52, in _ldap_function_call
    result = func(*args,**kwargs)
ldap.DECODING_ERROR
ERROR: Error: 
```

Reply:
@droideck fixed!